.gitignore

```diff
@@ -1,3 +1,5 @@
 .DS_Store
 node_modules/
-config/
+env
```

Comment:
This code doesn't ever create or expect an "env" file in the root directory, why ignore it here?